source/partials/drupal-8-to-drupal-9-upgrade.md

```diff
@@ -13,7 +13,9 @@ Run the code below to set the Drupal core version to Drupal 9:
   git commit -m "upgrade to Drupal 9"
   ```
 
-Use the command `composer update` instead of `composer update drupal/core* -W` if you receive the error message "Your requirements could not be resolved to an installable set of packages." 
+1. Use the command `composer update` instead of `composer update drupal/core* -W` if you receive the error message below:
```

Comment:
@whitneymeredith Could this be a note, since the step should only be taken if X?

Reply:
@michellecolon-pantheon I could see it either way.